bin/release.sh

```diff
@@ -29,3 +32,17 @@ if [ -z "$RELEASE_URL" ]; then
     -X POST \
     https://api.github.com/repos/nasa/cumulus-ecs-task/releases
 fi
+
+set -o pipefail
+
+if [ "$VERSION_NUMBER" = "$PREVIOUS_VERSION" ]; then
+  echo "$VERSION_NUMBER already released"
+  exit 0
+fi
+
+if [ "$VERSION_NUMBER" != "$PREVIOUS_VERSION" ]; then
+  npm publish --access public
```

Comment:
Is publishing to NPM really necessary?

Reply:
No, I don't think we have any reason to do so.